src/TypeSpec.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -133,12 +133,16 @@ function getCSharpInputTypeKindByIntrinsicModelName(
             }
         case "int8":
             return InputPrimitiveTypeKind.SByte;
-        case "unit8":
+        case "uint8":
```

Comment:
Can we have any test case to catch similar issues?

Reply:
We will add UT later.